evobandits/src/evobandits.rs

```diff
@@ -158,13 +158,20 @@ impl EvoBandits {
         self.genetic_algorithm.set_bounds(bounds);
         self.genetic_algorithm.validate();
 
+        if simulation_budget < self.genetic_algorithm.population_size {
+            panic!(
+                "simulation_budget must be at least population_size ({})",
+                self.genetic_algorithm.population_size
+            );
+        }
```

Comment:
```suggestion
        assert!(
            simulation_budget >= self.genetic_algorithm.population_size,
            "simulation_budget must be at least population_size ({})",
            self.genetic_algorithm.population_size
        );
```
Wouldn't this be semantically nicer?

Reply:
Yes. Did not think about that because [Python assertions can be bypassed](https://stackoverflow.com/questions/61826628/what-is-the-advantage-if-any-of-using-assert-over-an-if-else-condition) which means I tend to avoid them. [Rust's assert statement](https://doc.rust-lang.org/std/macro.assert.html) is safe though.